microsoft/PowerToys
# [common, shortcutguide] move d2d files from common to scg

## Summary of the Pull Request

Move the `d2d` code out of the common lib and back to the shortcut guide that is the only module using it.

## PR Checklist
* [x] Applies to https://github.com/microsoft/PowerToys/issues/7843

## Validation Steps Performed

_How does someone test & validate?_

Do a clean build, verify that the solution builds correctly. Run PowerToys.exe and verify the Shortcut Guide works as expected.
